firewood/src/storage/buffer.rs

```diff
@@ -790,42 +726,21 @@ mod tests {
         mut_store.write(0, &hash).unwrap();
         assert_eq!(mut_store.id(), STATE_STORE_ID);
 
-        // wal should have no records.
-        assert!(disk_requester.collect_ash(1).unwrap().is_empty());
-
         // get RO view of the buffer from the beginning.
         let view = mut_store.get_view(0, hash.len() as u64).unwrap();
         assert_eq!(view.as_deref(), hash);
 
         // Commit the change. Take the delta from cached store,
         // then apply changes to the CachedStore.
-        let (redo_delta, wal) = mut_store.delta();
+        let (redo_delta, _) = mut_store.delta();
```

Comment:
It looks like the second return value of `delta` is only ever used in tests. Should we remove it?